lib/context.js

```diff
@@ -50,22 +74,24 @@ const proto = module.exports = {
       res: '<original node res>',
       socket: '<original node socket>'
     };
-  },
+  }
 
   /**
    * Similar to .throw(), adds assertion.
    *
-   *    this.assert(this.user, 401, 'Please login!');
+   *    ctx.assert(ctx.user, 401, 'Please login!');
    *
    * See: https://github.com/jshttp/http-assert
    *
-   * @param {Mixed} test
+   * @param {Mixed} value
    * @param {Number} status
    * @param {String} message
+   * @param {Mixed} opts
    * @api public
    */
-
-  assert: httpAssert,
+  assert(value, status, msg, opts) {
```

Comment:
problem with this version is that you can no longer do:

```js
this.assert.equal(1234, 1234, 'Were not equal to 1234')
```

this wasn't documented though... and i never actually used it. it would be a breaking change though. https://github.com/jshttp/http-assert/blob/master/index.js